### PR TITLE
Mint's Signup page clone

### PR DESCRIPTION
**Pull request by leolpaz**
On this project we cloned the form from Mint's signup page, the form is responsive although it doesn't lead anywhere, we worked on the appearance of the form, we also made the nav-bar and footer to look like the ones on mint's page.

The page wasn't made to be responsive and works best in 1366x768 resolution with the firefox browser.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mint's sign up page</title>
+    <link rel="stylesheet" href="style.css">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,300;0,400;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,600;1,700;1,800;1,900&display=swap"
+        rel="stylesheet">
+    <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
```
Comment (descholar-ceo) [inline index.html]: Don't load scripts in head, do it at the end of body, for the sake of performance of your website

File: index.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mint's sign up page</title>
+    <link rel="stylesheet" href="style.css">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,300;0,400;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,600;1,700;1,800;1,900&display=swap"
```
Comment (descholar-ceo) [inline index.html:10]: For the sake of integrity validation, you should add crossorigin attribute to this `link` tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+                        </label>
+                        <p class="call">Standard call, messaging or data rates may apply.</p>
+                        <label class="inputs" for="password"> Password
+                            <input class="input-space" type="password" name="password" id="password">
+                        </label>
+                        <label class="inputs" for="confirm-password"> Confirm password
+                            <input class="input-space" type="password" name="confirm-password" id="confirm-password">
+                        </label>
+                    </fieldset>
+                    <button type="button" class="create-account"><span class="iconify" data-icon="bx:bxs-lock"
```
Comment (descholar-ceo) [inline index.html]: You should make this button of type  `submit`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+            <div class="qb">
+                <img src="./icons/qb.png" alt="qb">
+                quickbooks
+            </div>
+            <div class="turbotax">
+                <img src="./icons/turbotax.png" alt="turbotax">
+                turbotax
+            </div>
+        </div>
+    </section>
```
Comment (descholar-ceo) [inline index.html]: It would be better if this section is moved to the `main`.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+}
+
+.tittle-icons {
+  display: flex;
+  align-items: flex-end;
+  flex-wrap: nowrap;
+  margin-bottom: 1%;
+  width: 22%;
+  justify-content: space-evenly;
+  cursor: pointer;
```
Comment (descholar-ceo) [inline style.css]: You have already used this `cursor: pointer;` on line 32 and 55, so you are repeating yourselves, this is a bad practice. Advice: You can use a on class which has this property and apply it wherever you need this property.

File: README.md [hunk 6]
```diff
@@ -1 +1,38 @@
...
+
+# HTML Forms
+
+> On this project we make a clone of mint's sign up page as seen here: [mint](https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26ivid%3De042f6ed-04cd-42de-b457-5559d72b605f%26adobe_mc%3DMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1603397849%26ivid%3De042f6ed-04cd-42de-b457-5559d72b605f)
+
+![screenshot](https://github.com/gasb150/mint.com-s-signup-page-clone/blob/signup-page-clone/app_screenshot.png?raw=true)
+
+We made a clone of mint's sign up form using HTML and CSS, the page has a functional form, a mock navbar section and a mock footer, the page was made with the 1366x768 resolution in mind and might not be responsive in other resolutions.
+
+## Live Server
```
Comment (descholar-ceo) [inline README.md]: This title, you can just use `Live demo` not `live server`

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @leolpaz and @gasb150 :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to use sementic HTML, and you tried your best to use best practices in your CSS codes
## :warning: However we need to make few changes to mark this project approved, read them below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# Required Changes
- [x] The create account button should be of type submit [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png)
- [x] Currently the footer of your clone looks messy, try to add some spacing and margins to make it looks like the original
Your clone : 
Screenshot from 2020-10-23 19-37-41
Original : 
Screenshot from 2020-10-23 19-38-54
- [x] Fix all the comments I added directly in your codes

# Optional changes
- [x] Your commit messages should be in imperative mode, something like modify to solve linters reviews in css code instead of modified to solve linters reviews in css code, refer [to this article for mor details](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)
- [ ] On password field, you can add a lock icon
Your clone : 
Screenshot from 2020-10-23 20-13-06
Original : 
Screenshot from 2020-10-23 20-12-53
- [x] Your clone looks like the original but to make it even better, you should try to increase the height of input fields, to match the original

- [ ] Don't include scripts in head, all of scripts should be put on the bottom of the body of your html content

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :warning: 
### Good job @leolpaz and @gasb150 .
:+1: It's obvious you've put in a lot in this project and that's commendable.
#### However, there's still a few required changes to be made before this project can be approved.

# [Required] :hammer_and_wrench: 

- [ ] The logos should be horizontally aligned with their corresponding text

Original:
original_logos

Yours:
logos

- [ ] You should properly align the Create Account button

Original:
original_create

Yours:
create_btn

#### Implement the changes and submit for another review

Happy Coding :computer: 

Feel free to reach out to me 
slack `@Elijah Ayandokun`
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by Temmarie — CHANGES_REQUESTED** [review]
## Changes Required! :recycle: 
## Hello Team! 
  
### You have done well so far :clap: However, you did not finish all the changes requested by the previous TSE. Kindly work on them  submit them for another code review.

- [x] The logos should be horizontally aligned with their corresponding text. 

##### Original:
original

##### Yours:
Screenshot (10)


__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Comment by mimipeshy** [comment]
### **STATUS : APPROVED :dancers: :tada:** 

Hi @leolpaz and @gasb150 congratulations on making all the required changes :fireworks: , your project is approved :clap :clap:  :clap: 

Good luck with your next projects! You may merge.


Happy coding !!
Peris Ndanu

**Pull request merged**